### PR TITLE
cas 295 - update about modal

### PR DESCRIPTION
**Pull request by germanurrus**
Screen Shot 2022-07-24 at 20 04 34
Mobile:
Screen Shot 2022-07-24 at 19 49 01
Note: disregard separator below beta label, appears when inspecting but is not showing on regular view

### CODE REVIEW

File: frontend/packages/client/src/components/WalletConnectModal.js [hunk 1]
```diff
@@ -33,25 +37,26 @@ export default function WalletConnectModal({
           style={{ borderBottom: 'none' }}
         >
           <div className="column p-0 is-flex flex-1">
-            <h2 className="is-size-4">Connect a wallet</h2>
+            <h2 className="is-size-4 has-text-weight-bold">Connect a wallet</h2>
```
Comment (germanurrus) [inline frontend/packages/client/src/components/WalletConnectModal.js:40]: unrelated change: fixes background color of footer, it was too dark

File: frontend/packages/client/src/components/WalletConnectModal.js [hunk 2]
```diff
@@ -33,25 +37,26 @@ export default function WalletConnectModal({
...
           </div>
           <div
             className={`column is-narrow px-0 has-text-right is-size-2 leading-tight cursor-pointer`}
             onClick={closeModal}
           >
-            &times;
+            <Close />
```
Comment (germanurrus) [inline frontend/packages/client/src/components/WalletConnectModal.js:46]: change icon to use the one from designs

**Review by dbslone — APPROVED** [review]
